docs/quickstart.md

```diff
@@ -4,7 +4,7 @@ This page provides some basic guidance on getting up and running with your first
 
 You'll need to install the following pre-requisites in order to build SAFE applications
 
-* The [.NET Core SDK 3.1](https://dotnet.microsoft.com/download/dotnet-core/)
+* The [.NET 5 SDK](https://dotnet.microsoft.com/download/dotnet/5.0)
 * [node.js](https://nodejs.org/) (>= 8.0)
 * [npm](https://www.npmjs.com/)
```

Comment:
might be worth adding a note in brackets that npm should be bundled with recent node.js versions